app/services/analytics_events.rb

```diff
@@ -665,8 +665,8 @@ def idv_address_submitted(
   end
 
   # User visited idv address page
-  def idv_address_visit
-    track_event('IdV: address visited')
+  def idv_address_visit(**extra)
```

Comment:
Can we document the new parameters these methods are receiving rather than relying on `extra` ?

Reply:
I started doing this manually and then decided it was time to actually learn about Rubocop cops. I'm working on a cop to enforce that these "enhanced" properties are added to all the methods they should be